src/components/Match/TeamTable.jsx

```diff
@@ -19,19 +26,20 @@ const TeamTable = ({
   direTeam = {},
   summable = false,
   hoverRowColumn = false,
+  loggedInId,
 }) => (
   <div>
     <Heading
       title={`${getTeamName(radiantTeam, true)} - ${heading}`}
       icon={<IconRadiant />}
     />
-    <Table data={filterMatchPlayers(players, 'radiant')} columns={columns} summable={summable} hoverRowColumn={hoverRowColumn} />
+    <Table data={filterMatchPlayers(players, 'radiant')} columns={columns} summable={summable} hoverRowColumn={hoverRowColumn} highlightUser={highLightUser(loggedInId, players, true)} />
```

Comment:
I think highlightUser should be highlightFn and passed as a function that the table runs each row through to determimne its' styling.

e.g. `const getHighlightFn = (loggedInId) => (row) => return row.account_id === loggedInId`